pyomo/dae/tests/test_flatten.py

```diff
@@ -208,7 +292,997 @@ def c_rule(m, t):
             self.assertIn(self._hashRef(ref), ref_data)
 
 
-    # TODO: Add tests for Sets with dimen==None
+class TestFlatten(TestCategorize):
+
+    def _hashRef(self, ref):
+        if not ref.is_indexed():
+            return (id(ref),)
+        else:
+            return tuple(sorted(id(_) for _ in ref.values()))
+
+    def _model_1(self):
```

Comment:
Can you provide more descriptive model names (maybe incorporate some of the subsequent comment into the function name; possibly something like `_model1_1D_sets`)

Reply:
Done